pkg/printers/internalversion/describe.go

```diff
@@ -2589,6 +2589,7 @@ func describeNode(node *api.Node, nodeNonTerminatedPodsList *api.PodList, events
 		printLabelsMultiline(w, "Labels", node.Labels)
 		printAnnotationsMultiline(w, "Annotations", node.Annotations)
 		printNodeTaintsMultiline(w, "Taints", node.Spec.Taints)
+		w.Write(LEVEL_0, "Unschedulable:\t%v\n", node.Spec.Unschedulable)
```

Comment:
move down a line to keep the metadata info next to each other

Reply:
(or move creation timestamp up by labels and annotations)

Reply:
make sense, move create timestamp closer to labels.